.github/workflows/github-actions.yml

```
@@ -57,20 +69,27 @@ jobs:
             ${{ secrets.ENV_FILE }}
             EOF
             chmod 600 .env
             
-            docker stop interview-be-app || true # 컨테이너가 실행 중이면 중지
-            docker rm interview-be-app || true # 기존 컨테이너 삭제
-            docker rmi ${{ secrets.DOCKER_USERNAME }}/interview-be-service || true # 기존 이미지 삭제
+            # 최신 이미지 pull & app 교체
+            docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
+            docker compose pull app
+            docker compose up -d app
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)